docs/common-reference-docs/3-configuring-rules.md

```diff
@@ -63,29 +61,29 @@ Note: * indicates a required field.
 
 To find the token details, please refer to this article: [Find Token Details](./finding-token-details)
 
-**chainId***: Enter the chain ID that your token is on.The chainID is a unique ID assigned to each chain(L1 and L2) - For example 1 is the the chainID for Ethereum Mainnet 137 is the chainID of Polygon Mainnet. You can find the chainIds here: [Chainlist](https://chainlist.org/)
+**chainId***: The chainID is a unique ID assigned to each chain(L1 and L2) - For example `1` is the the chainID for Ethereum Mainnet while `137` is the chainID of Polygon Mainnet. You can find more chainIDs on [Chainlist](https://chainlist.org/).
 
-**type***: Select the token type of your token. For example, for EVM based chains, it can ERC-20, ERC-721, ERC-1155, etc.
+**type***: Select the token type of your token. For example, for EVM based chains, it can be ERC-20, ERC-721, ERC-1155, etc. For non-EVM chains, it can be BEP-20, etc. If you are not sure about the token type, you can use the [Find Token Details](./finding-token-details) article to find the token type.
```

Comment:
BEP-20 - Binance is an EVM-compatible chain, please remove this as an example. For non-EVM chains, you can give examples of Solana, Flow, etc